modules/phase_field/src/ics/MultiSmoothSuperellipsoidIC.C

```diff
@@ -141,13 +158,13 @@ MultiSmoothSuperellipsoidIC::computeSuperellipsoidCenters()
       Real ran2 = _random.rand(_tid);
       Real ran3 = _random.rand(_tid);
 
-      newcenter(0) = _bottom_left(0) + ran1*_range(0);
-      newcenter(1) = _bottom_left(1) + ran2*_range(1);
-      newcenter(2) = _bottom_left(2) + ran3*_range(2);
+      newcenter(0) = _bottom_left(0) + ran1 * _range(0);
+      newcenter(1) = _bottom_left(1) + ran2 * _range(1);
+      newcenter(2) = _bottom_left(2) + ran3 * _range(2);
 
-      for (unsigned int j = 0; j < i; j++)
+      for (unsigned int j = start; j < i; j++)
```

Comment:
Because a `newcenter` is not added to the `_centers` array until after it has passed all checks it can never be compared to itself by this loop.